scss/components/_side-nav.scss

```diff
@@ -30,6 +30,15 @@
   &:last-child {
     padding-bottom: 0;
   }
+
+  &.is-disabled {
```

Comment:
I tried removing the inner span because that sounds like the cleaner way to go, but looks like the link style is providing some padding. Is that a bug? What do you suggest?

<img alt="screenshot from 2016-04-11 16-32-00" src="https://cloud.githubusercontent.com/assets/509703/14445601/51e76bde-0003-11e6-865c-517ba83a3b0b.png" width="200px">